qutip/hsolve.py

```diff
@@ -31,8 +31,10 @@
 #    OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ###############################################################################
 """
-This module provides exact solvers for a system-bath setup using the
-hierachical method.
+2016-03-9: This module has been deprecated.
```

Comment:
The qutip.hsolver has not yet been part of a release, right? Or was it already here in 3.1? If not, then you should not have deprecation message. Just deleting this module is fine. We dont guarantee api stability until after release.